### PR TITLE
Rw drink results cards

### LINKED ISSUE #8: Create a displayResults component

### LINKED ISSUE #9: Create cards component to display drink recipes

### PR DESCRIPTION
**Pull request by rwilliams659**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features

### Detailed Description / What does this PR do?
This adds the Results and DrinkCards components. All CSS styling has been added including media queries for the mobile view as well.
Test files have been added with basic tests for rendering. Router will need to be added to those, and additional tests will need to be added once we actually have our props being passed down. 

There is also currently a test data file for Results that I was using in the Results component to test how everything was getting displayed. We can delete this later. 

### Files modified:
Results and DrinkCard files - js, test files, and css 

### Has This Been Tested?
- [ ] No
- [X] Yes

### Are there specific methods for testing?
Traditional React testing right now. 

### What are the relevant tickets?
resolve #8 
resolve #9 

### Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/DrinkCard/DrinkCard.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.DrinkCard {
+  border: 1px solid gray;
+  border-radius: 0.8em;   
+  box-shadow: 0.3em 0.3em 1.2em #F65C78;
+  background-color: white; 
+}
+
+.DrinkCard:hover {
+  transform: scale(1.05);
```
Comment (relyt4me) [inline src/components/DrinkCard/DrinkCard.css:9]: This is great

File: src/components/DrinkCard/DrinkCard.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+.DrinkCard {
```
Comment (tarynmartin) [inline src/components/DrinkCard/DrinkCard.css:1]: we should see about making the box shadow all around to make it look like it's backlit by neon lights

**Review by tarynmartin — APPROVED** [review]
Looks great!

File: src/components/Results/Results.test.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    const recipeCard1Name = screen.getByText('3-Mile Long Island Iced Tea');
+    const recipeCard2Name = screen.getByText('Ace');
+
+    expect(recipeCard1Name).toBeInTheDocument(); 
+    expect(recipeCard2Name).toBeInTheDocument(); 
+
+  });
+
+  it('should display an error message if there are no results to display', () => {
```
Comment (relyt4me) [inline src/components/Results/Results.test.js:36]: Good call

**Review by relyt4me — APPROVED** [review]

**Pull request merged**